### PR TITLE
Formatting stylization

### PR DESCRIPTION
**Pull request by phg49389**
Went through the code base and cleaned it up. Organized imports, changed a lot of quote marks to apostrophes, did a few small bug fixes, and increased the display size of the Description field being rendered by the templates.

### CODE REVIEW

File: app/forms.py [hunk 1]
```diff
@@ -35,43 +40,43 @@ def _safe(form, field):
...
     def render_to_html(self):
-        return render_template("forms/generic.html", fields=self._fields.iteritems())
+        return render_template('forms/generic.html', fields=self._fields.iteritems())
 
 
 # WTForm objects that are used in rendering. Each Field in this object corresponds to the user-input columns in the DB
 class RegularPostForm(RenderableForm):
-    id = HiddenField("", [validators.DataRequired(), validators.Length(max=8)])
-    submitters_username = HiddenField("", [validators.DataRequired(), validators.Length(max=50)])
+    id = HiddenField('', [validators.DataRequired(), validators.Length(max=8)])
```
Comment (e-jameson) [inline app/forms.py]: Can you use a different name that doesn't collide with the built in keyword id? PEP8 would say to use `id_` or a different name altogether.

File: app/forms.py [hunk 2]
```diff
@@ -35,43 +40,43 @@ def _safe(form, field):
...
     categories = SelectMultipleField('Categories:', [validators.DataRequired()],
                                      choices=get_category_list(return_list_of_tuples=True))
-    submit = SubmitField("Submit")
+    submit = SubmitField('Submit')
 
 
 class ExternalPosterForm(RenderableForm):
-    id = HiddenField("", [validators.DataRequired(), validators.Length(max=8)])
-    submitters_username = StringField("Email of external poster:", [validators.DataRequired(),
+    id = HiddenField('', [validators.DataRequired(), validators.Length(max=8)])
```
Comment (e-jameson) [inline app/forms.py]: Same here

File: app/models.py [hunk 3]
```diff
@@ -50,7 +47,7 @@ def __init__(self, username, first, last, email, phone):
         self.is_admin = False
 
     def __repr__(self):
-        return "<Contact '%s'>" % self.first_name + " " + self.last_name
+        return "<Contact '%s'>" % self.first_name + ' ' + self.last_name
```
Comment (e-jameson) [inline app/models.py]: its kinda weird using both `%s` notation and `+ ' ' +`. Could you move the name combination to the line above and use `%s` for both?

**Comment by phg49389** [comment]
I've gone through and done a lot of code refactoring; organized the imports, replaced most of the quotes with apostrophes, removed some old comments and print statements, redid some logic, redid a few naming conventions, and added an __all__ dunder to controller.py to simplify the imports, fixed a few display errors, and added Josiah's new version of the logout method.

**Pull request merged**